### DISCUSSION TITLE
Cannot change server thumbnail, and cannot find the source file for it

### DISCUSSION THREAD

**Discussion by M-Stenzel** [mastodon/mastodon#20526] [Q&A]
Hi, this was not reported before, and I am not sure whether this is a bug.
As an  admin I cannot change the server thumbnail picture (latest release of mastodon) on this page https://mastodon.server.org/admin/settings/edit .
I can delete the entry, and add a new entry (upload new picture), but no change when verifying that the thumbnail was changed.

Now I wanted to change the picture by hand in the appropriate file, but I cannot find the file, that serves https://mastodon.server.org/about/more.
I checked for all files in /home/mastodon/live , but was not able to find this file.

When I look after the source code (https://mastodon.server.org/about/more) in the browser I see that the server description was updated recently (which is correct) but the entry for "public-account-header__image" was not changed (remained the same as in the initial setup).

Any clues?

P. S. No errors are shown in the browser console...

**Comment by MrGeneration** [comment]
I guess you mean the default image didn't change in the about page?
Depending on your systems load it may take to refresh.

Even on my relatively low load instance I've seen something in between 2-10 minutes.
Probably due to caching. Same applies to custom CSS btw.

**Reply by M-Stenzel** [reply]
Hey, you are absolutely right.
It took a couple of hours at my instance, but, no problem at all.
Cache sounds reasonable.

Thank you,

Martin.
